freqtrade/strategy/strategy.py

```diff
@@ -20,32 +20,19 @@ class Strategy(object):
     """
     This class contains all the logic to load custom strategy class
     """
-    __instance = None
-
-    DEFAULT_STRATEGY = 'default_strategy'
-
-    def __new__(cls) -> object:
-        """
-        Used to create the Singleton
-        :return: Strategy object
-        """
-        if Strategy.__instance is None:
-            Strategy.__instance = object.__new__(cls)
-        return Strategy.__instance
-
-    def init(self, config: dict) -> None:
+    def __init__(self, config: dict = {}) -> None:
```

Comment:
Using mutable default variables lead to probably unexpected side effects, see: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments